src/developer-quickstart.md

```diff
@@ -31,7 +31,7 @@ A script is runnable bytecode on the chain which can call contracts to perform s
 | script    | ❌                                   | ❌                    | ❌                           | ❌                        |
 | library   | ✅ (via a contract or predicate) | ❌ | ❌                           | ✅                       |
 
-See [the chapter on program types](../sway-program-types/index.md) for more information.
+See [the chapter on program types](https://fuellabs.github.io/sway/v0.19.2/sway-program-types/index.html) for more information.
```

Comment:
Should we use the latest version here? We sadly can't use `latest` in the link above but we can use `mater`:
```suggestion
See [the chapter on program types](https://fuellabs.github.io/sway/master/sway-program-types/index.html) for more information.
```